shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/domain/repository/ResetPasswordRepository.kt

```diff
@@ -0,0 +1,9 @@
+package co.nimblehq.blisskmmic.domain.repository
+
+import co.nimblehq.blisskmmic.data.model.ResetPasswordMeta
+import kotlinx.coroutines.flow.Flow
+
+interface ResetPasswordRepository {
+
+    fun reset(email: String): Flow<ResetPasswordMeta>
```

Comment:
Naming suggestion for better clarity, because at first, I thought it is for resetting email 🙇‍♂️
```suggestion
    fun resetPassword(with email: String): Flow<ResetPasswordMeta>
```